wordpress-mobile/WordPress-iOS
# Adds a Tooltip for the Create Button

#13787

Adds the tooltip prompt to the Create Button. The Notice is repurposed to show this particular tooltip. It hasn't been fully genericized to place a tooltip anywhere (the arrow is in a fixed position), but can be positioned using a `sourceView` which is used to anchor the tooltip.

<a href="https://user-images.githubusercontent.com/3250/80668256-9df72480-8a5e-11ea-95e6-8422e8cb75d5.png"><img src="https://user-images.githubusercontent.com/3250/80668256-9df72480-8a5e-11ea-95e6-8422e8cb75d5.png" width="300"></a>
<a href="https://user-images.githubusercontent.com/3250/80668262-a51e3280-8a5e-11ea-9027-ba8ddc039cc0.png"><img src="https://user-images.githubusercontent.com/3250/80668262-a51e3280-8a5e-11ea-9027-ba8ddc039cc0.png" width="600"></a>

To test:
- Open the app to any Blog Details page (My Sites > Select a blog)
- Notice the tooltip that shows above the Create Floating Action Button

PR submission checklist:
